black-mulberry-core/src/main/java/com/black/mulberry/core/exception/UserEmailConflict.java

```diff
@@ -0,0 +1,23 @@
+package com.black.mulberry.core.exception;
+
+public class UserEmailConflict extends RuntimeException {
+
+    public UserEmailConflict() {
+    }
```

Comment:
I don't recommend using constructors that don't take arguments as message. lets delete them.